.github/workflows/ci.yml

```diff
@@ -13,3 +13,8 @@ jobs:
     uses: ./.github/workflows/build-and-run-tests-posix.yml
     with:
       platform: macos-latest
+
+  build-and-run-tests-on-windows:
+    uses: ./.github/workflows/build-and-run-tests-windows.yml
+    with:
+      platform: windows-latest
```

Comment:
I'm completely opposed to our current setup with `ci.yml`. What is the purpose of it AT ALL?
Yes, you need this kind of workaround on Gitlab, but not on Github.

This is a great opportunity to remove it completely.

Reply:
> Which workaround do you mean?

Why do we have a `ci.yml` at all?

Usually you have such a setup at Gitlab, where you can not have multiple 'root' workflows that all run in parallel - you need one dummy workflow that calls another workflow file multiple times.
With Github that is not needed.

Reply:
I believe I set it up this way to easily call the same "Posix" workflow on two different platforms. Adding a somewhat different Windows workflow came naturally. But if you have a better solution, go ahead.

Reply:
> I believe I set it up this way to easily call the same "Posix" workflow on two different platforms

That can also be done within one workflow file :-)

Reply:
> That can also be done within one workflow file :-)

Then please. go ahead. I have no desire at all to mess with the CI setup, and you usually see this in the quality of the result... :)